translate/rus.json

```diff
@@ -303,8 +303,8 @@
 "play every":"играть каждый",
 "player":"игрок",
 "Please consider whitelisting [TCEC_URL] in your ad blocker --- or send a donation.":"Пожалуйста, добавьте в белый список [TCEC_URL] в вашем блокировщике рекламы --- или отправьте пожертвование.",
-"plies":"",
-"plies/s":"скорость",
+"plies":"полуходы",
+"plies/s":"полуходов/с",
```

Comment:
Some new translations, a ply can be considered as 'half-move', that is how I translated it